integration-templates/woocommerce/types.ts

```diff
@@ -0,0 +1,163 @@
+export interface WooCommerceOrder {
```

Comment:
I often do the same, `Order` vs. `SomeApiOrder`.

Other ideas:
- `Order` vs. `_Order` | `_${API_PROVIDER}_Order`
- `Order` vs. `Internal_Order` | `Internal_${API_PROVIDER_Order`

I'm personally OK with `Order` vs. `WooCommerceOrder` FWIW

Reply:
I think WooCommerceOrder is fine but it's usually how we differentiate stuff in `nango.yaml` when you have multiple integrations. It might be safer to use `Internal_Order` indeed, and it needs to apply to all the types inside the file. Otherwise no other ideas :/